### ISSUE TITLE
Fix accessibility issues found by axe-core

### ISSUE THREAD

**Issue by robertknight** [hypothesis/lms#1396]
Adding [runtime accessibility tests](https://github.com/hypothesis/lms/pull/1395) using axe-core found several failures.

In [the PR](https://github.com/hypothesis/lms/pull/1395) these failures have been flagged by `// FIXME-A11Y` comments in the test files followed by a skipped test (`it.skip("should pass a11y checks")`).

**Comment by lyzadanger** [comment]
This is done.
